core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -132,7 +132,7 @@ class TaskManager(taskManagerResources: SystemResources = TaskManagerDefaults.de
                   logDirectory: Option[Path]                 = None,
                   scheduler: Scheduler                       = TaskManagerDefaults.defaultScheduler,
                   simulate: Boolean                          = false,
-                  sleepMilliseconds: Int                     = 250
+                  sleepMilliseconds: Int                     = 10
```

Comment:
I wonder if this should be exposed via config since we keep tweaking it?  I worry setting it this low will lead to a thread is largely busy-waiting to see if it should do something when the DAG is not that dense.